src/main/frontend/components/block.cljs

```diff
@@ -301,11 +302,10 @@
        [:img.rounded-sm.relative
         (merge
          {:loading "lazy"
-          :src     src
+          :src     image-src
```

Comment:
Thanks for the fix; it works fine for me on the desktop version, too. should figure out why the assets protocol href was left here.

Some relevant issues and PRs:

- https://github.com/logseq/logseq/issues/7021
- https://github.com/logseq/logseq/pull/6475